string yaml_path
---
bool success
string message